docs/source/boards/boards.rst

```diff
@@ -71,6 +73,9 @@ Espressif
     ESP32-S2-Saola-1 <ESP32-S2-Saola-1>
     ESP32-C3-DevKitM-1 <ESP32-C3-DevKitM-1>
 
+.. note:: 
+   Just few boards are described in this documentation page, for more information about other Espressif boards please refer to [HW reference section](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/hw-reference/index.html) in ESP-IDF Programming Guide.
```

Comment:
Maybe we change to:
Only a few development boards are described on this documentation page. For more information about other Espressif boards please refer to the [Development Boards](https://www.espressif.com/en/products/devkits) session on the Espressif website.

Reply:
@pedrominatel You are right, changed. Thanks for review.